### DISCUSSION TITLE
Can i set the state externaly, not just from a react component?

### DISCUSSION THREAD

**Discussion by ivan-angjelkoski** [pmndrs/jotai#1943] [Q&A]
lets say we have a counter atom, and usualy we will set the state using the `useAtom` hook and then `setCount`.
can i set the count outside of a react component, but still rerender the componts that use it? 
Ex:

```js
const count = atom(1)

function increaseCount() {
    count.setState(3)
}

**Comment by dai-shi** [comment]
You can use store api for that use case: https://jotai.org/docs/core/store

It's an advanced api, and not recommended for typical use cases.
